src/installer/test/Assets/TestProjects/PortableApp21/PortableApp21.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project>
+
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
```

Comment:
Do we need to explicitly do the imports for some reason instead of just setting the `Sdk` attribute on the project?